packages/responder/src/al-responder-client.ts

```diff
@@ -82,6 +83,43 @@ export class AlResponderClientInstance {
         });
     }
 
+    /**
+     * List all playbooks
+     * GET
+     * /v1/{account_id}/playbooks
+     * https://responder.mdr.global.alertlogic.com
+     *
+     * @param accountId AIMS Account ID
+     * @returns Playbook list
+     *
+     * @remarks
+     *
+     * */
+    async getAllPlaybooks(accountId: string, parameters: AlPlaybookRequest = {}): Promise<AlResponderPlaybook[]> {
+        let playbooks = [];
+        let marker = undefined;
+
+        do {
+            const params = {
```

Comment:
do you really need to defined this constant on each iteration, isn's better to have a let params with the parts that do not change and merge the marker on the iteration itself?

Reply:
ok